bazel/_logging_threshold_test_main.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import tempfile
+import sys
+import subprocess
+
+LOGGING_ERROR_THRESHOLD = 50
+LOGGING_OUT_THRESHOLD = 20
```

Comment:
Switching to a zero logs or "skipped" model is a blocker for merging this PR.

Reply:
we are skipping the tests with warning for now